cobc/codegen.c

```diff
@@ -7100,6 +7100,7 @@ output_java_call (struct cb_call *p)
 	char *last_dot;
 	char *method_name;
 	const char *class_name;
+	char return_type_signature[32];
 	char method_signature[2048] = "(";
```

Comment:
use `COB_MINI_BUFF` and `COB_NORMAL_BUFF` for those and down use `strncat` with the `_MAX` defines